Automattic/wp-calypso
# Jetpack Cloud: Add new scripts for development and docker

#### Changes proposed in this Pull Request

* Introduce an `npm run start-jetpack-cloud` script to start a development server.
* Introduce an `npm run docker-jetpack-cloud` script to run a production Docker instance.

#### Testing instructions

* Checkout this branch.
* Run `npm run start-jetpack-cloud` and verify the dev server starts, and when it builds, Jetpack Cloud is accessible at http://jetpack.cloud.localhost:3000/
* Run `npm run docker-jetpack-cloud` after you have built the docker image with `npm run build-docker` and verify the `jetpack-cloud-production` Docker instance starts.

Related Asana card: 1156570014567299-as-1159371543675233
